community/assessments/guide/README.md

```diff
@@ -69,10 +71,10 @@ PR, citing the presentation issue number to add the self-assessment to
 [assessments/projects](/community/assessments/projects)
 under its own folder.  The ticket may then be closed after merged in.
 
-### Growing projects
+### More detailed project assessment
```

Comment:
I am wondering if we should define this more detailed assessment with a specific name so that we can easily distinguish this process from the "abbreviated assessment" that projects at an early maturity stage may be interested in (e.g. "Comprehensive security review", "Joint Security Review")